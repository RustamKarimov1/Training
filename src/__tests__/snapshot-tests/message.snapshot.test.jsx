import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import ErrorBoundary from '../../components/error-boundary/error-boundary';

configure({ adapter: new Adapter() });

describe('<ErrorBoundary />', () => {
    it('should render correctly if hasError state is false', () => {
        const ERRORBOUNDARY = shallow(<ErrorBoundary />);
        ERRORBOUNDARY.setState({ hasError: true });
        expect(shallowToJson(ERRORBOUNDARY)).toMatchSnapshot();
    });
});
