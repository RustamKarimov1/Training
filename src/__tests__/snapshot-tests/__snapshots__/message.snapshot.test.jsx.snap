// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorBoundary /> should render correctly if hasError state is false 1`] = `
<h1>
  Something went wrong.
</h1>
`;
